src/config/config.ts

```diff
@@ -0,0 +1,55 @@
+import fs from "fs";
+import path from "path";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export interface LanguageConfig {
+  language: string;
+  dockerfilePath: string;
+  runCommand: string;
+}
+
+export const LANGUAGE_CONFIGS: Record<string, LanguageConfig> = {
+  typescript: {
+    language: "typescript",
+    dockerfilePath: path.join(
+      __dirname,
+      "../../supportedLangaugeDockerfiles/typescript/Dockerfile",
+    ),
+    runCommand: "/app/.hxckr/run.sh",
+  },
+  rust: {
+    language: "rust",
+    dockerfilePath: path.join(
+      __dirname,
+      "../../supportedLangaugeDockerfiles/rust/Dockerfile",
```

Comment:
```suggestion
      "../../supportedLanguageDockerfiles/rust/Dockerfile",
```

Reply:
@daviroo can pinpoint what changed here, can you elaborate?